test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -488,6 +494,27 @@ public void testIncludeAllColumnsAsMetadataFalse_mcBlank() throws Exception {
   }
 
   @Test
+  public void testUniqueKey() throws Exception {
+    // mpaggi here
+    // db.uniqueKey=productid is wrong, should be db.uniqueKey=productid:int
+    // type accepted are int, string, timestamp, date, time, and long
```

Comment:
How about """
Value if unique id cannot be "productid", because that is missing type. The value
has to be something like "productid:int".
"""